bridge/build.rs

```diff
@@ -0,0 +1,17 @@
+use std::process::Command;
+
+fn main() {
+    // rerun build script if bridge contract has changed.
+    // without this cargo doesn't since the bridge contract
+    // is outside the crate directories
+    println!("cargo:rerun-if-changed=../contracts/bridge.sol");
+    let exit_status = Command::new("solc")
+        .arg("--abi")
+        .arg("--bin")
+        .arg("--output-dir").arg("../compiled_contracts")
+        .arg("--overwrite")
+        .arg("../contracts/bridge.sol")
```

Comment:
should we also add argument to optimize compiled code?

Reply:
definitely

Reply:
done